modules/openapi-generator/src/main/resources/python/model_utils.mustache

```diff
@@ -1512,10 +1515,16 @@ def get_oneof_instance(cls, model_kwargs, constant_kwargs, model_arg=None):
 
         try:
             if not single_value_input:
-                oneof_instance = oneof_class(**model_kwargs, **constant_kwargs)
+                if '_spec_property_naming' in constant_kwargs and constant_kwargs['_spec_property_naming']:
```

Comment:
This looks good, one can use the shorter `constant_kwargs.get('_spec_property_naming')` which returns None by default which is falsey